mysql-test/suite/rpl/t/bug84415.test

```diff
@@ -0,0 +1,265 @@
+-- source include/master-slave.inc
```

Comment:
--source include/have_innodb.inc, above master-slave.inc. Or if the test is SE-agnostic, remove ENGINE=... clauses